### DISCUSSION TITLE
Academic affiliation verification failed due to the distance to the college

### DISCUSSION THREAD

**Discussion by pjosalgado** [community/community#47547] [GitHub Education] [answered]
Hi Community!

Just requested access to the GitHub Student Developer Pack using a declaration from my college but it was denied with the following explanation: `You appear not to be near any campus location for the school you have selected. If you are a distance learner then your school-provided academic affiliation documentation must state so.`

The declaration sent contained a indication that the course enrolled it's of a long distance's type. The only point it's the declaration is in portuguese, since the college is in Brazil. At the document, this was indicated with "modalidade EaD" phrase on it, wich means "distance learning modality".

If this was not sufficient, whats is expected the declaration to have?

Thanks!

**Comment by unknowVariable** [comment, marked as answer]
I'm sorry to hear that your request for access to the GitHub Student Developer Pack was denied. Based on the reason provided, it seems that GitHub requires proof that you are a distance learner, and that the documentation you provided did not clearly state that you are a distance learner.

If the documentation you provided did include the phrase "modalidade EaD," which indicates distance learning, it's possible that there was a miscommunication or error on GitHub's end. In this case, I would recommend reaching out to GitHub's support team directly to explain the situation and provide any additional information that may be helpful.

If you need to provide additional documentation to prove that you are a distance learner, you could consider asking your college to provide a letter or certificate that clearly states your status as a distance learner. This could include details such as your enrollment in a specific distance learning program, the start and end dates of the program, and any other relevant details.

I hope this helps, and I wish you the best of luck in resolving this issue.

**Reply by pjosalgado** [reply]
> I'm sorry to hear that your request for access to the GitHub Student Developer Pack was denied. Based on the reason provided, it seems that GitHub requires proof that you are a distance learner, and that the documentation you provided did not clearly state that you are a distance learner.
> 
> If the documentation you provided did include the phrase "modalidade EaD," which indicates distance learning, it's possible that there was a miscommunication or error on GitHub's end. In this case, I would recommend reaching out to GitHub's support team directly to explain the situation and provide any additional information that may be helpful.
> 
> If you need to provide additional documentation to prove that you are a distance learner, you could consider asking your college to provide a letter or certificate that clearly states your status as a distance learner. This could include details such as your enrollment in a specific distance learning program, the start and end dates of the program, and any other relevant details.
> 
> I hope this helps, and I wish you the best of luck in resolving this issue.

Hi @alixconstantin, thanks for the response!
How can I get in touch with the GitHub's support team about this subject?

**Reply by unknowVariable** [reply]
To contact GitHub's support team about an issue, you can follow these steps:

1. Go to the GitHub support page (https://support.github.com/)
2. Scroll down to the "Contact us" section and click on the "Contact us" button
3. Select the appropriate category that best matches your issue from the dropdown list
4. Follow the instructions to provide a detailed description of the issue and any relevant information, including screenshots or error messages if applicable
5. Once you have submitted your support request, you should receive a confirmation email from GitHub with a ticket number. GitHub's support team will review your request and respond to you as soon as possible.

Alternatively, you can also reach out to GitHub's support team by sending an email to [support@github.com](mailto:support@github.com). Make sure to include a detailed description of the issue and any relevant information in your email.

If I helped you, please select me as your answer, I'm doing a contest with a friend to have the most answers selected 🦊

**Reply by pjosalgado** [reply]
I'll open a ticket about this subject. Thank you for the assistance!

**Reply by unknowVariable** [reply]
@pjosalgado  It was a pleasure ! gitHub support should normally find a solution for you quickly 😊

**Reply by samus-aran** [reply]
We appreciate your assistance @alixconstantin. 👍  This type of inquiry we would not recommend directing to the support team as they are unable to provide manual reviews.

**Comment by samus-aran** [comment]
Hi @pjosalgado 👋 

All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants.

But we're here to assist 👍 

For remote student applicants: please ensure that your evidence indicates that you are a remote student. 
If your evidence is not in English you can submit a pre-translated copy of the evidence along side your original. This may help you indicate that you are a remote learner.

Hope this helps and let us know how you get on!
